WebRoot/js/zimbraMail/mail/view/ZmMailMsgView.js

```diff
@@ -1831,6 +1831,7 @@ ZmMailMsgView.prototype._renderMessageBody1 = function(params, part) {
         // calendar part in ICS format
         else if (ct === ZmMimeTable.TEXT_CAL) {
             content = ZmMailMsg.getTextFromCalendarPart(bp);
+            content = AjxStringUtil.htmlEncode(content);
```

Comment:
ZmMailMsg.getTextFromCalendarPart  is being called from here only.
More ever, getTextFromCalendarPart  gives me the text content out of a calendar part, so it's good to not use it there.
More ever, the context here is while rendering of message, when the malicious script of 
Description field is getting executed inside hidden Iframe. And, not in other cases.
So, better to take care of encoding here, itself.

Thanks.